src/styles/main.scss

```diff
@@ -0,0 +1,284 @@
+$c-seconds: rgb(0, 255, 45);
+$c-sec-axis: rgb(0, 255, 45);
+$c-sec-glow: rgb(0, 255, 45);
+
+$c-minutes: hsl(131, 75%, 50%);
+$c-min-axis: hsl(131, 75%, 50%);
+$c-min-glow: hsl(131, 75%, 50%);
+
+$c-hours: hsl(131, 50%, 45%);
+$c-hour-glow: hsl(131, 50%, 45%);
+$c-hour-axis: hsl(131, 50%, 45%);
+
+$c-lines: hsla(130, 50%, 50%, 0.68);
+$c-circles: rgba(68, 196, 68, 0.32);
+
+$size-glow: 0.5;
+$size-glow-blur: 0.1;
+
+$size-main: 75;
+$size-circle: 9;
+$size-pad: 70;
+$size-second: 62;
+$size-minute: 63.5;
+
+$size-main-ls: #{$size-main}vh;
+$size-main-pt: #{$size-main}vw;
+
+$size-br-radius: 1vh;
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background: conic-gradient(
+    rgba(255, 255, 255, 0.8),
+    rgba(0, 0, 0, 0.2),
+    rgba(255, 255, 255, 0.8)
+  );
+  height: 100vh;
+  margin: 0;
+  display: flex;
+  justify-content: center;
+}
+
+.watch {
+
+  & > * {
+    opacity: 0.68;
+  }
+
+  margin: auto;
+  @media (orientation: landscape) {
+    width: $size-main-ls;
+    height: $size-main-ls;
+  }
+  @media (orientation: portrait) {
+    width: $size-main-pt;
+    height: $size-main-pt;
+  }
+  position: relative;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  border-radius: 50%;
```

Comment:
Add to `circle` mixin (with `$size` argument)

Reply:
fix this comment

Reply:
Fix this too